Makefile

```diff
@@ -1,9 +1,10 @@
 SDK ?= "iphonesimulator"
 DESTINATION ?= "platform=iOS Simulator,name=iPhone X"
 PROJECT := Analytics
-XC_ARGS := -scheme $(PROJECT) -workspace $(PROJECT).xcworkspace -sdk $(SDK) -destination $(DESTINATION) GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES
-XC_BUILD_ARGS := ONLY_ACTIVE_ARCH=NO
-XC_TEST_ARGS := GCC_GENERATE_TEST_COVERAGE_FILES=YES
+TEST_PROJECT := $(PROJECT)Tests
+XC_ARGS := -workspace $(PROJECT).xcworkspace -sdk $(SDK) -destination $(DESTINATION) GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES
+XC_BUILD_ARGS := -scheme $(PROJECT) ONLY_ACTIVE_ARCH=NO
+XC_TEST_ARGS := -scheme $(TEST_PROJECT) GCC_GENERATE_TEST_COVERAGE_FILES=YES RUN_E2E_TESTS=$(RUN_E2E_TESTS) WEBHOOK_AUTH_USERNAME=$(WEBHOOK_AUTH_USERNAME)
```

Comment:
what happens if the environment (e.g. a local machine) does not have this (`RUN_E2E_TESTS`) set? does it just skip the tests and still work?

Reply:
I tested locally and it seems to work as expected!

Reply:
Right, it should only run it when the `RUN_E2E_TESTS` environment variable is `true`